Home/home.html

```diff
@@ -60,7 +61,25 @@ <h4 class="grey">Racquets</h4>
                     <h1>Badminton</h1>
                 </div>
             </div>
-
+            <h2 class="centered-text grey">Featured Categories</h2>
+            <div class="featured-categories">
+                <div class="featured-item">
+                    <img src="../Images/rq-3.png" alt="">
```

Comment:
It's good practice to give value to the `alt` attribute as it helps in accessibility. You could give value to this image and all the following one's.

Reply:
sure. thank you for the compliments